optimum/intel/ipex/modeling_base.py

```diff
@@ -414,7 +501,7 @@ def forward(
             position_ids = attention_mask.long().cumsum(-1) - 1
             position_ids.masked_fill_(attention_mask == 0, 1)
             if past_key_values:
-                position_ids = position_ids[:, -1].unsqueeze(-1)
+                position_ids = position_ids[:, -1].unsqueeze(0)
```

Comment:
why this modification ? 
```suggestion
                position_ids = position_ids[:, -1].unsqueeze(-1)
```